### PR TITLE
adding Toolbar js object

### LINKED ISSUE #218: Toolbar: cannot get instance of MDComponent
``` jsx
<Toolbar ref={tb => this.tb = tb}>
          <Toolbar.Row>...</Toolbar.Row>
</Toolbar>
```

``` js
console.log(this.tb.MDComponent) // undefined
```
I need that instance to handle 'MDCToolbar:change' event. But it's always undefined. Do I miss something?

### PR DESCRIPTION
**Pull request by prateekbh**
Fixes #218

### CODE REVIEW

**Comment by prateekbh** [comment]
@filoozom I'll be travelling for next couple of days, can you please publish the new version after merging this?

**Comment by filoozom** [comment]
Sure, was just about to open a pull request for this too 😄

File: Toolbar/Toolbar.jsx [hunk 1]
```diff
@@ -20,6 +20,12 @@ class Toolbar extends MaterialComponent {
       "flexible-default-behavior"
     ];
   }
+  componentDidMount() {
+    this.MDComponent = new MDCToolbar(this.control);
```
Comment (filoozom) [inline Toolbar/Toolbar.jsx]: Shouldn't you add ```this.MDComponent.listen("MDCToolbar:change", this._onChange);``` there? To have something like the ```IconToggle``` component.

Comment (prateekbh) [inline Toolbar/Toolbar.jsx]: doing it right away

Comment (prateekbh) [inline Toolbar/Toolbar.jsx]: added

**Comment by filoozom** [comment]
Still need to bind ```_onChange``` I think.

**Comment by prateekbh** [comment]
done

**Pull request merged**